### DISCUSSION TITLE
Some Ideas for upgrades to Elementor Pro

### DISCUSSION THREAD

**Discussion by Aryehhirsc** [elementor/elementor#15349] [Feature Request]
As someone who has built multiple websites using Elementor pro, I put together some ideas that would help with design and functionality. I understand some of these may have already been brought up to staff, but here it goes.

1. **Header** - Add the ability to shrink the sticky header menu on scroll & ability to swap out full-size logo to smaller logo to fit shrink size.
2. **Header** - More **_mobile_** menu draw or drop-down design/functionality options. Currently, these are extremely limited. I have something specific in mind, but I'm not sure if I'm allowed to post links here.
3. **Scroll to top** - Scroll to the top widget (I know there are ways to add this, but it would be nice to have a simple to use widget or setting)
4. **Hotspots** - In the latest update, hotspots have been introduced (I love it). It would be nice to have the ability to create hotspots on slideshows as well.
5. **Hotspots** - Ability to stretch/cover image
6. **Swap Fonts** The ability to preload and swap fonts on load through Elementor with no additional plugin would be incredible. (this is critical for my SEO)
7. **Footer** - The ability to add before footer templates. To be specific, I put a form on every page before the footer, but on some pages, I also include an about us section. Being able to control that through the template feature would be great.

I have more suggestions, but I guess I'll leave that for another post.

**Comment by crispee2000** [comment]
I definitely agree with the mobile menu thing - it is way limited and takes too many third party add-ons to get what i need to do sometimes.
